live-examples/html-examples/inline-text-semantics/css/kbd.css

```diff
@@ -0,0 +1,6 @@
+kbd {
+  background: #f5f5f5;
+  border-style: solid;
+  border-width: 1px 2px 2px 1px;
+  padding: 0px 1px 0px 1px;
```

Comment:
Nit: Code style: no units with 0 values. Can also be shortened -  `padding: 0 1px;`